packages/tailwindcss/src/property-order.ts

```diff
@@ -75,6 +75,7 @@ export default [
   'translate',
   '--tw-translate-x',
   '--tw-translate-y',
+  '--tw-translate-z',
```

Comment:
This was missing, but we do have it for `--tw-scale-z` and `--tw-rotate-z` so I figured why don't we just add this.